lib/mp4/transmuxer.js

```diff
@@ -299,6 +302,18 @@ collectDtsInfo = function (track, data) {
     if (track.timelineStartInfo.pts === undefined) {
       track.timelineStartInfo.pts = data.pts;
     }
+
+    if (track.minSegmentPts === undefined) {
+      track.minSegmentPts = data.pts;
```

Comment:
Doesn't need to be resolved in this PR but I think having a notion of a "segment" here is probably not good.